main_server/local_server/install_wizard/installer.js

```diff
@@ -737,10 +747,10 @@ function gmScriptLoadCompletion()
 *   \returns a new XMLHTTPRequest object.                                                   *
 ********************************************************************************************/
     
-function BMLT_Installer_AjaxRequest(
+function BMLT_Installer_AjaxPostRequest(
```

Comment:
All usages of this function were for `POST`, so I simplified it by removing the `method` option.